stdlib/bytes.gr

```diff
@@ -605,5 +625,4 @@ export let rec make = (n: Number) => {
 /**
  * An empty byte sequence
  */
-export let empty = make(0)
-
+export let empty = make(0)
```

Comment:
Did it remove the newline at the end of the file?

Reply:
Hmmm, I've seen this a couple of times now.   It may have and I think it needs to keep it.

Reply:
This is fixed in the next push